app/src/components/UI/Pages/VendorDashboard.tsx

```diff
@@ -0,0 +1,64 @@
+import { Container, Card, Icon, Header } from "semantic-ui-react";
+import HeaderBar from "../Molecules/HeaderBar/HeaderBar";
+import styles from "./vendordashboard.module.css";
+
+const VendorDashBoard: React.FC = () => {
+  return (
+    <Container className={styles.wrapper}>
+      <HeaderBar logout />
+      <Header as={"h2"} className={styles.header}>
+        Vendor Dashboard
+      </Header>
+      <Card.Group className={styles.cardGroup}>
+        <Card className={styles.card}>
+          <Icon name="setting" size="huge" className={styles.icon} />
+          <Card.Content className={styles.content}>
+            <Card.Header>Customize Vendor Page</Card.Header>
+            <Card.Meta>Updated in 2016</Card.Meta>
```

Comment:
Looks great. This update notice makes no sense here or in the trends card.

Reply:
Ok.